### DISCUSSION TITLE
Add setting for multiple sites/languages

### DISCUSSION THREAD

**Discussion by tomfischerNL** [digitalwithyou/craft-facebook-conversion#24] [Ideas]
Hi @Diewy,

We have a craft website with 2 sites. Both sites are on another domain, and we use 2 facebook pixels.
Maybe it would be nice to have a setting per language/site to add the pixel and access token, so the multiple sites can have different settings.

How do you think about that?

Thanks in advance,

**Comment by Diewy** [comment]
Good point. I'm going to work on the Craft 4 version first, but I'll see what I can do to support this asap.

**Comment by tomfischerNL** [comment]
Thanks for the quick support ;)
